deployment/k8s-config/add-project/skaha-add-project-keel-dev.yaml

```diff
@@ -13,9 +13,10 @@ spec:
         image: images.canfar.net/skaha-system/add-project:1.0
         imagePullPolicy: Always
         # TODO: automate the setting of this in the calling script
+        # 5th argument is project quota in GB
         command: ["/usr/bin/add-project"]
         # args: project-dir-name, owner-userid, project-group-name, read-only or read-write
-        args: ["test-project", "majorb", "skaha-users", "read-write"]
+        args: ["test-project", "majorb", "skaha-users", "read-write", "1000"]
```

Comment:
Instead of the comments on line 16, add quota-gb to the comment on line 18

Reply:
Right. Done.